src/style.css

```diff
@@ -1,3 +1,67 @@
+:root {
+  --main-color: #00acdc;
+}
+
 body {
+  font-family: Roboto, sans-serif;
+  margin: 0;
+}
+
+.header {
+  font-size: 12px;
+  font-style: normal;
+  font-weight: 500;
+  text-transform: uppercase;
+  display: flex;
+  justify-content: space-between;
+  padding: 0 50px;
+  align-items: center;
+  height: 60px;
+}
+
+.nav {
+  display: flex;
+  height: 100%;
+  cursor: pointer;
+}
+
+.nav__ul {
+  list-style-type: none;
+  display: flex;
+  padding: 0;
   margin: 0;
 }
+
+.nav__link {
+  text-decoration: none;
+  color: black;
+  display: flex;
+  height: 100%;
+  margin-right: 20px;
+  align-items: center;
+}
+
+.nav__link--last {
+  margin-right: 0;
+}
+
+.nav__link:hover {
+  color: var(--main-color);
+}
+
+.is-active {
+  color: var(--main-color);
+  position: relative;
+}
+
+.is-active::after {
+  content: "";
+  display: block;
+  height: 4px;
+  width: 100%;
+  border-radius: 8px;
+  background-color: var(--main-color);
+  color: var(--main-color);
```

Comment:
You can remove color from after. Only background color affects the color of this 'underline'.